.github/workflows/handler-test.yml

```
@@ -40,9 +40,19 @@ jobs:
         id: validate
         working-directory: ./tests/public-install
         env:
-          AWS_DEFAULT_REGION: us-west-2
+          AWS_DEFAULT_REGION: us-east-2
         run: terraform validate -no-color
 
+      - name: Terraform Apply
+        id: apply
+        working-directory: ./tests/public-install
+        run: terraform apply -input=false -no-color
+
+      - name: Terraform Destroy
+        id: destroy
+        working-directory: ./tests/public-install
+        run: terraform destroy -force -input=false -no-color
```

I think we also need `-auto-approve` here:

```suggestion
        run: terraform apply -auto-approve -input=false -no-color
```